### PR TITLE
bugfix/18960-dumbbell-data-labels

### LINKED ISSUE #18960: Data labels for Dumbell of multiple series are collecting at the same serie.
Am not sure what has changed internally here, but it looks like an issue which could be more general than just this case I observed.

Highcharts GPT was particularly useful in creating an MRE!

#### Expected behaviour
Data labels are attached to their respective series

#### Actual behaviour
All data labels are attached to the same series

You can downgrade the imported scripts from 11 to 10 to see the previous behavior.

#### Live demo with steps to reproduce
https://jsfiddle.net/kluvin/a1Lg4k75/

#### Product version
11.0.0 >
 
#### Affected browser(s)
FF, not tested in Chrome.

### PR DESCRIPTION
**Pull request by jakubSzuminski**
Fixed #18960, a regression in v11, wrong placement for data labels in dumbbell series.

The issue was the change of `ColumnRangeSeries.prototype.translate()` function which was used in `DumbellSeries` as a `setShapeArgs` function.

**Demo before fix:** https://jsfiddle.net/kluvin/a1Lg4k75/
**Demo after fix:** https://jsfiddle.net/BlackLabel/e9fa0ck8/

### CODE REVIEW

**Review by karolkolodziej — APPROVED** [review]
Please add test as Torstein mentioned

**Comment by jakubSzuminski** [comment]
This dumbbell sample should pick up on this issue as the `dataLabels` are now shown.

File: samples/highcharts/demo/dumbbell/demo.js
Comment (TorsteinHonsi) [inline samples/highcharts/demo/dumbbell/demo.js:1]: This is the main demo that we display in our web pages, https://www.highcharts.com/demo/dumbbell

I don't think we should add data labels here, as it kind of clutters up the nice clean appearance of the chart. A Google Images search confirms that this chart type is more often used without data labels. We have a sample for `highcharts/series-dumbbell/styled-mode-dumbbell` that is not part of the website demos, we could add the labels there instead.

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]

File: samples/highcharts/demo/dumbbell/demo.js
Comment (jakubSzuminski) [reply samples/highcharts/demo/dumbbell/demo.js:1]: You're right. I'm sorry, my mistake. I've now enabled dataLabels for the `highcharts/series-dumbbell/styled-mode-dumbbell` sample.

**Pull request merged**